css/styles.css

```diff
@@ -444,3 +451,174 @@ header {
 #contact-me {
   background: #fff;
 }
+
+@media (max-width: 767px) {
+  .menu-list {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .menu-btn {
+    display: none;
+  }
+
+  header {
+    padding: 20px 10%;
+  }
+
+  .menu-list li {
+    font-size: 1.5rem;
+    font-weight: 500;
+    line-height: 20px;
+    text-align: center;
+    margin-left: 32px;
+    color: #344563;
+  }
+
+  .fab {
+    font-size: 2.6rem;
+  }
+
+  .work-card {
+    flex-direction: row;
+    padding: 3%;
+    align-items: center;
+    justify-content: space-evenly;
+  }
+
+  .work-card-pair {
+    flex-direction: row-reverse;
+  }
+
+  .work-card img {
+    width: 40%;
+    height: auto;
+    margin-right: 22px;
+    margin-bottom: 0;
+  }
+
+  .work-card-data {
+    padding: 24px;
+    align-self: flex-start;
+  }
+
+  .work-card-info {
+    margin-bottom: 24px;
+  }
+
+  .work-card-description {
+    font-size: 1.6rem;
+    margin-bottom: 24px;
+  }
+
+  .about-myself-top {
+    flex: 0 1 35%;
+  }
+
+  .about-myself-top h1,
+  .about-myself-top p,
+  .about-myself-top .fab {
+    margin-bottom: 24px;
+  }
+
+  .about-myself-skill-list {
+    flex: 0 1 55%;
+  }
+
+  .skill-list-menu-open-list {
+    flex-direction: row;
+    justify-content: flex-start;
+  }
+
+  .skill-list-menu-open li {
+    max-width: 125px;
+    margin-right: 12px;
+    flex-direction: column;
+  }
+
+  .skill-icon {
+    margin-right: 0;
+    margin-bottom: 12px;
+  }
+
+  .contact-me-form {
+    padding: 50px 15%;
+    align-items: center;
+    background: #6070ff url(../media/contact-form-bg-desktop.png) 50% 50% no-repeat;
+  }
+
+  .contact-me-form ul {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .contact-me-form li {
+    width: 100%;
+  }
+
+  .form-input-box {
+    border-radius: 0;
+  }
+
+  .form-button {
+    margin: 0 auto;
+  }
+
+  #introduction {
+    padding: 0 17%;
+    border-bottom-left-radius: 12rem;
+    background: #fff url(../media/header-bg.png) 90% 50% no-repeat;
+  }
+
+  #introduction h1 {
+    font-size: 4.8rem;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  #introduction p {
+    font-size: 2rem;
+    line-height: 28px;
+  }
+
+  #about-myself {
+    flex-direction: row;
+    padding: 114px 10%;
+    justify-content: space-between;
+    align-content: center;
+  }
+}
+
+@media (min-width: 1200px) {
+  .work-card-tittle {
+    font-size: 4rem;
+    line-height: 5.2rem;
+  }
+
+  .work-card-info h2 {
+    font-size: 1.8rem;
+    line-height: 2.4rem;
+    font-weight: 600;
+  }
+
+  .contact-me-form {
+    padding: 50px 30%;
+  }
+
+  .work-card-info h3 {
+    font-size: 1.8rem;
+    line-height: 2.4rem;
+    font-weight: 600;
+  }
+
+  #introduction {
+    background: #fff url(../media/header-bg.png) 50% 50% no-repeat;
+  }
+
+  #introduction p {
+    max-width: 80%;
+  }
+}
```

Comment:
I suggest you stick to the design as much as possible, This is your design.
Please compare, check the difference between both decoration positions and make the necessary adjustment
![Screenshot (33)](https://user-images.githubusercontent.com/52497246/121437333-090d6d00-c97a-11eb-86cd-50bb94e6ec12.png)

Figma design 
![Screenshot (34)](https://user-images.githubusercontent.com/52497246/121437464-4e319f00-c97a-11eb-9f7e-580f36fd268b.png)

Reply:
Adjustments were made so it looks more like the original.